template.ufs-noahMP.namelist.gdas_gts

```diff
@@ -0,0 +1,80 @@
+&run_setup
```

Comment:
Delete this file, and use the template.ufs-noahMP.namelist.gdas one, and add code to sub in the model time step as is done for the template.coupler.res file. Also, add the same code to the other forcing files (*gefs and *gswp3)

Reply:
Done